dist/fixto.js

```diff
@@ -1,7 +1,5 @@
-/*! fixto - v0.5.0 - 2016-06-16
```

Comment:
if you happen to push more commits, please do not add this file until releasing. i usually have a release commit. Until then it should not be in the history. It is also making the pr confusing.

The version number also will be incremented while preparing this file. Currently all these are maintained manually.